internal/core/data/event.go

```diff
@@ -236,6 +238,15 @@ func updateEventPushDate(id string, ctx context.Context) error {
 func putEventOnQueue(evt models.Event, ctx context.Context) {
 	LoggingClient.Info("Putting event on message queue")
 
+	//Re-marshal JSON content into bytes.
+	if clients.FromContext(clients.ContentType, ctx) == clients.ContentTypeJSON {
+		data, err := json.Marshal(evt)
+		if err != nil {
+			LoggingClient.Error(fmt.Sprintf("error marshaling event: %s", evt.String()))
+			return
+		}
+		evt.Bytes = data
+	}
 	evt.CorrelationId = correlation.FromContext(ctx)
```

Comment:
Do we wanna move this line above the remarshalling in the case that it fails the correlationId will be in the error output?

Reply:
@rsdmike I will include this with the subsequent export-distro PR so I don't have to re-acquire an approval.